### ISSUE TITLE
Unable to handle task responses to actions

### ISSUE THREAD

**Issue by blakehilliard** [DMTF/Redfishtool#55]
If a system returns 202 to a reset action, so that the action is tracked with a task, redfishtool fails:

```
$ python3 redfishtool.py -S Always -r host -u user -p pass Systems -I 1 reset ForceOff
   redfishtool: Transport: processing response status codes
$ echo $?
5
```

**Comment by rdilip42** [comment]
still i am facing same issue. 
 redfishtool: Transport: processing response status codes
   redfishtool: raw: Error sending POST to resource, aborting
#DB5: raw: operation returned with error: rc=5
#DB5: #DB4:Main: subcommand returned with error: rc=5
# Main: Error: rc=5
#DB5:    Response status code:202

**Comment by rdilip42** [comment]
#DB5: Main: subcmd: raw, subCmdArgs:['raw', 'POST', '/redfish/v1/Systems/Partition1/Actions/ComputerSystem.Reset']
#DB5: Main: verbose=5, status=0, user=administrator, password=HPE1nnov, rhost=ah-061b-rmc.chf.rdlabs.hpecorp.net
#DB5: Main: token=None, RedfishVersion=v1, Auth=Basic, Timeout=10
#DB5: Main: prop=None, Id=None, Match=None:None, First=False, -1=False, Link=None
#DB5: Main: gotIdOptn=False, IdOptnCount=0, gotPropOptn=False, gotMatchOptn=False, gotEntriesOptn=False
#DB5: Main: 2nd-Level Collection Member reference options: -i<id>=None, -m<match>=None:None, -l<link>=None -all=False
#DB5: Main: 2nd-level Collection Member parsing: gotIdLevel2Optn=False, gotMatchLevel2Optn=False, IdLevel2OptnCount=0
#DB5: Main: configFile=, Secure=Always, waitNum:waitTime=1:3, Degug=00000000
#DB5: Main: Headers=None
#DB5: Main: options parsed.  Now lookup subcommand and execute it
#DB5: runSubCmd: subcmd: raw
#DB5: runSubCmd: argvs:  ['raw', 'POST', '/redfish/v1/Systems/Partition1/Actions/ComputerSystem.Reset']
#DB5: runSubCmd: found SubCmd: raw in table. executing
#DB4: RawMain:  subcommand: raw
#DB5: raw: operation=POST, args=['POST', '/redfish/v1/Systems/Partition1/Actions/ComputerSystem.Reset']
#DB5: raw: runOperation: operation: POST
#DB5: raw:runOperation:  args:  ['POST', '/redfish/v1/Systems/Partition1/Actions/ComputerSystem.Reset']
#DB5: raw:runOperation: found Oper: raw in table. executing
#DB4: raw:POST: in raw
#DB4: raw: POST: method:POST path:/redfish/v1/Systems/Partition1/Actions/ComputerSystem.Reset
#DB5: Transport.rftProcessRequest: method=POST, baseUrl=https://ah-061b-rmc.chf.rdlabs.hpecorp.net/redfish, rpath=/redfish/v1/Systems/Partition1/Actions/ComputerSystem.Reset
#DB5: Transport.rftProcessRequest: apiType=2
#DB5: Transport.ProcessRequest: url=https://ah-061b-rmc.chf.rdlabs.hpecorp.net/redfish/v1/Systems/Partition1/Actions/ComputerSystem.Reset
#REQUEST: Transport:SendRecv:    POST https://ah-061b-rmc.chf.rdlabs.hpecorp.net/redfish/v1/Systems/Partition1/Actions/ComputerSystem.Reset
   redfishtool: Transport: processing response status codes
   redfishtool: raw: Error sending POST to resource, aborting
#DB5: raw: operation returned with error: rc=5
#DB5: #DB4:Main: subcommand returned with error: rc=5
# Main: Error: rc=5
#DB5:    Response status code:202
#DB5:    Response headers: {'Date': 'Wed, 18 Dec 2019 17:42:01 GMT', 'Server': 'Apache', 'X-Frame-Options': 'SAMEORIGIN', 'X-XSS-Protection': '1', 'Strict-Transport-Security': 'max-age=63072000; includeSubDomains', 'Location': '/redfish/v1/TaskService/Tasks/80f0354cea734d36bb970d1a1e32870e/Monitor', 'OData-Version': '4.0', 'Cache-Control': 'no-cache,no-store,must-revalidate', 'ETag': 'W/"6c934ddfe6a6ccfde8645004a7ffe8fa"', 'Link': '<http://redfish.dmtf.org/schemas/Task.v1_1_0.json>; rel=describedby', 'Content-Security-Policy': "default-src 'self' 'unsafe-inline' 'unsafe-eval';                                         style-src 'self' 'unsafe-inline';                                         img-src 'self' data:;                                         connect-src wss: https:;", 'X-Content-Security-Policy': "default-src 'self'", 'X-WebKit-CSP': "default-src 'self'", 'X-Content-Type-Options': 'nosniff', 'Pragma': 'no-cache', 'Keep-Alive': 'timeout=5, max=100', 'Connection': 'Keep-Alive', 'Transfer-Encoding': 'chunked', 'Content-Type': 'application/json'}
#DB5: rfCleanup:Cleaningup session: None

**Comment by billdodd** [comment]
You should really open a new issue. This issue was closed over a year ago.

That said, it looks like you are using an old version of Reffishtool that does not include the fix from #58.

From your output above:

```
redfishtool: Transport: processing response status codes
```

That line of output does not exist in the current version of the tool. It was changed in the #58 fix for this status 202 issue. Please retrieve the latest version of the tool and try your test again.
